1_space_odyssey/js/camilla-9000/app.js

```diff
@@ -356,3 +357,71 @@ console.groupEnd();
  Add 2 New planets of your choosing to the front of the Array : https://nineplanets.org
 */
 
+/*
+ 7. Android Object REFACTOR:::::
+
+ Level 1::::::::::::::
+ Refactor (change) all the variables you have in place that initializes the android into an android object.
+ - Your android needs a head, upperBody, lowerBody & [ anyThingElseYouCanDreamOf ].
+ - Feel free to create any new properties that will further describe your android [ anyThingYouCanDreamOf ].
+ - next update your application to use this newly refactored code
+
+ tip: the console.log that prints the variables of the android will need to be updated
+
+ Level 2::::::::::::::
+ Refactor the planets array to be an array of objects, each object must be its own planet.
+  - next update your application to use this newly refactored code
+*/
+
+
+var android = [ {
+  name: "Camilla-9000",
```

Comment:
nice, close so you can create one object to represent "android" and in that have more objects like this

```js
var obj = {
     head: {
        ...
     },
     upperBody: {
        ...
     },
     lowerBody: {
        ...
     },
}
```